src/app/api/admin/policies/apiPolicies.html

```diff
@@ -98,8 +98,26 @@ <h3 class="gravitee-policy-path" ng-if='path === "/"'>{{path}}</h3>
                     </div>
                   </md-card-content>
                   <div layout="column">
-                    <ng-md-icon icon="delete" style="fill: #757575" size="18px" ng-if='"api-key" !== policy.policyId' class="gravitee-card-close" ng-click="apiPoliciesCtrl.removePolicy($index, path, $event)"></ng-md-icon>
-                    <ng-md-icon icon="mode_edit" style="fill: #757575" size="18px" class="gravitee-card-close" ng-click="apiPoliciesCtrl.editPolicyDescription($index, path, $event)"></ng-md-icon>
+                    <ng-md-icon icon="close" style="fill: #757575" size="18px" ng-if='"api-key" !== policy.policyId' class="gravitee-card-close" ng-click="apiPoliciesCtrl.removePolicy($index, path, $event)">
+                      <md-tooltip md-direction="right">
+                        Remove policy
+                      </md-tooltip>
+                    </ng-md-icon>
+                    <ng-md-icon icon="mode_edit" style="fill: #757575" size="18px" class="gravitee-card-close" ng-click="apiPoliciesCtrl.editPolicyDescription($index, path, $event)">
+                      <md-tooltip md-direction="right">
+                        Edit policy description
+                      </md-tooltip>
+                    </ng-md-icon>
+                      <ng-md-icon icon="done" ng-if="policy.enabled && 'api-key' !== policy.policyId" style="fill: #437536" size="18px" class="gravitee-card-close" ng-click="apiPoliciesCtrl.switchPolicyEnabled($index, path, $event)">
```

Comment:
Display `Disable policy` on the tooltip may confuse uses I think.

What do you think if we display a message like "Policy is enabled. Click to disable it." ?